src/main/java/com/concamap/controllers/user/UserController.java

```diff
@@ -90,6 +94,42 @@ public ModelAndView showCreateForm(@PathVariable("id") int id, @SessionAttribute
 
 //    @PostMapping("/users/{id}/create")
 //    public ModelAndView savePost(@ModelAttribute("post")Post post, @PathVariable("id") int id){
+//        ModelAndView modelAndView = new ModelAndView("post/create");
+//        MultipartFile multipartFile = ;
+//        String title, content, fileName, fileUpload;
+//        File file;
 //
 //    }
+
+    @GetMapping("/users/{id}/posts/{anchor-name}/edit")
+    public ModelAndView showEditForm(@PathVariable("id") int id, @PathVariable("anchor-name") String anchorName, @SessionAttribute("categoryList") List<Category> categoryList){
+        Post post = postService.findExistByAnchorName(anchorName);
+        ModelAndView modelAndView = new ModelAndView("post/edit");
+        modelAndView.addObject("post", post);
+        modelAndView.addObject("categoryList", categoryList);
+        return modelAndView;
+    }
+
+    @PostMapping("/users/{id}/posts/edit")
+    public ModelAndView updatePost(@PathVariable("id") int id,@ModelAttribute("post")Post post, @SessionAttribute("categoryList") List<Category> categoryList){
+        postService.save(post);
+        ModelAndView modelAndView = new ModelAndView("post/edit");
+        modelAndView.addObject("post", post);
+        modelAndView.addObject("categoryList", categoryList);
+        return modelAndView;
+    }
+
+    @GetMapping("/users/{id}/posts/{anchor-name}/delete")
+    public ModelAndView showDeleteForm(@PathVariable("id") int id, @PathVariable("anchor-name") String anchorName){
+        Post post = postService.findExistByAnchorName(anchorName);
+        ModelAndView modelAndView = new ModelAndView("post/delete");
+        modelAndView.addObject("post", post);
+        return modelAndView;
+    }
```

Comment:
1. Trả về Redirect View không phải Model And View.
2. Chưa có logic xoá Post, yêu cầu bổ sung